### PR TITLE
Add strings necessary for Critical Emails UI features

### PR DESCRIPTION
**Pull request by maxxcrawford**
See Figma for additional context: https://www.figma.com/file/aw8PCsC4nVUr4rSQAPU7fm/Critical-Emails?node-id=210%3A405

### CODE REVIEW

File: en/app.ftl [hunk 1]
```diff
@@ -321,6 +321,10 @@ faq-question-acceptable-use-answer-measure-rate-limit = Rate-limiting the number
 #   $attrs (string) - specific attributes added to external links
 faq-question-acceptable-use-answer-b-html = Please review our <a href="{ $url }" { $attrs }>Terms of Service</a> for more information.
 
+faq-question-promotional-email-blocking-question = What is promotional email blocking?
+faq-question-promotional-email-blocking-answer = Relay Premium subscribers can enable promotional email blocking. This feature will forward you important emails, such as receipts, password resets and confirmations, while still blocking marketing messages. There is a slight risk that an important message could still be blocked, so we recommend that you not use this feature for very important places like your bank. If an email is blocked, it cannot be recovered.
```
Comment (peiying2) [inline en/app.ftl:325]: - `Relay Premium` is a brand.
- Remove the `,` before `while still blocking marketing message.`

File: en/app.ftl [hunk 2]
```diff
@@ -321,6 +321,10 @@ faq-question-acceptable-use-answer-measure-rate-limit = Rate-limiting the number
...
+faq-question-detect-promotional-question = How does Relay detect if an email is Promotional or not?
+faq-question-detect-promotional-answer = Many emails are sent with “header” metadata to indicate that they are from list-based automated tools. Firefox Relay detects this header data so it can block these emails.
```
Comment (peiying2) [inline en/app.ftl:327]: `Firefox Relay` is a brand.

File: en/app.ftl [hunk 3]
```diff
@@ -394,6 +398,22 @@ profile-filter-category-option-domain-based-aliases-v2 = Custom aliases
...
+## Alias Promotional Email Blocking (displayed on the profile page)
+profile-promo-email-blocking-title = What emails do you want to block?
+# Block all emails sent to a speciic alias
+profile-promo-email-blocking-option-all = All
+# Block promotional emails sent to a speciic alias
+profile-promo-email-blocking-option-promotionals = Promotionals
+# Allow/forward all emails sent to a speciic alias
+profile-promo-email-blocking-option-none = None
+profile-promo-email-blocking-description-all = { -brand-name-relay } is blocking all emails sent to this alias.
+profile-promo-email-blocking-description-promotionals = { -brand-name-relay } will attempt to block promotional emails, while still forwarding emails like receipts and shipping information.
```
Comment (peiying2) [inline en/app.ftl:410]: Remove `,` before `while still forwarding emails... `

**Review by peiying2 — CHANGES_REQUESTED** [review]
Brand names need to be coded the Fluent way; plus punctuation related issue.

File: en/app.ftl (hunk 1 above)
Comment (Vinnl) [reply en/app.ftl]: I replaced the brand names in e95ec7b. The comma looked correct to me, so I didn't touch that and will defer to Maxx and Chris on that :)

File: en/app.ftl [hunk 4]
```diff
@@ -321,6 +321,10 @@ faq-question-acceptable-use-answer-measure-rate-limit = Rate-limiting the number
 #   $attrs (string) - specific attributes added to external links
 faq-question-acceptable-use-answer-b-html = Please review our <a href="{ $url }" { $attrs }>Terms of Service</a> for more information.
 
+faq-question-promotional-email-blocking-question = What is promotional email blocking?
+faq-question-promotional-email-blocking-answer = Relay Premium subscribers can enable promotional email blocking. This feature will forward you important emails, such as receipts, password resets and confirmations, while still blocking marketing messages. There is a slight risk that an important message could still be blocked, so we recommend that you not use this feature for very important places like your bank. If an email is blocked, it cannot be recovered.
+faq-question-detect-promotional-question = How does Relay detect if an email is Promotional or not?
```
[the remark this answers is not in the document]
Comment (Vinnl) [reply en/app.ftl]: Fixed in e95ec7b.

File: en/app.ftl (hunk 2 above)
Comment (Vinnl) [reply en/app.ftl]: Fixed in e95ec7b.

File: en/app.ftl (hunk 1 above)
Comment (peiying2) [reply en/app.ftl]: @maxxcrawford

File: en/app.ftl (hunk 3 above)
Comment (maxxcrawford) [reply en/app.ftl]: Revised in 0c5106c652527ae5b78cf3ac22ed24b14977fc5e

**Review by peiying2 — APPROVED** [review]
Thank you for confirming and for making the changes.

File: en/app.ftl [hunk 5]
```diff
@@ -394,6 +398,22 @@ profile-filter-category-option-domain-based-aliases-v2 = Custom aliases
 # Filter on Relay aliases that only forward critical emails.
 profile-filter-category-option-critical-only-aliases = Critical-only aliases
```
Comment (Vinnl) [inline en/app.ftl:399]: @maxxcrawford Hmm, looks like this string was already there, but it's the only place we refer to "critical" emails - maybe check with Chris?

**Pull request merged**